### PR TITLE
Additional detail in the solution to the "Doing a Dry Run" exercise

### PR DESCRIPTION
**Pull request by jyalim**
Simple edit adding detail to the "Doing a Dry Run" exercise in episode 5 on loops.

### CODE REVIEW

**Comment by gdevenyi** [comment]
Thanks for this addition, I like it. Can you please address the one small comment I have. Thanks!

File: _episodes/05-loop.md [hunk 1]
```diff
@@ -706,13 +706,16 @@ so she decides to get some coffee and catch up on her reading.
...
+> > is treated literally as part of a string rather than as a
+> > redirection instruction.
 > >
 > > The first version appends the output from the command `echo cat $datafile` 
 > > to the file, `all.pdb`. This file will just contain the list; 
 > > `cat cubane.pdb`, `cat ethane.pdb`, `cat methane.pdb` etc.
 > >
 > > Try both versions for yourself to see the output! Be sure to open the
-> > `all.pdb` file to view its contents.
+> > `all.pdb` file to view its contents.
```
[the remark this answers is not in the document]
Comment (jyalim) [reply _episodes/05-loop.md:718]: I removed the space in an amended commit, I think I got it?

[the remark this answers is not in the document]
Comment (gdevenyi) [reply _episodes/05-loop.md:718]: Hi, I still see the errant space here.

[the remark this answers is not in the document]
Comment (gcapes) [reply _episodes/05-loop.md:718]: @jyalim Did you force push the amended commit?

**Comment by deppen8** [comment]
Closed by #1251

**Pull request closed without merging**